lib/Scenes/Page/HomePage/Component/greeting_information.dart

```diff
@@ -2,6 +2,23 @@ import 'package:flutter/material.dart';
 import 'package:timeboxing/Shared/Extension/colors_style_extension.dart';
 import 'package:timeboxing/Shared/Extension/text_style_extension.dart';
 
+class GreetingsItem {
+  String username;
+  String greetingMessage;
+
+  GreetingsItem({
+    required this.username,
+    required this.greetingMessage,
+  });
+}
+
+final List<GreetingsItem> greetingitem = [
```

Comment:
use camelCase naming
```suggestion
final List<GreetingsItem> greetingItem = [
```